data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -19,11 +29,30 @@
             <column name="key_material" type="VARBINARY(1048576)">
                 <constraints nullable="false"/>
             </column>
+            <!-- if specified, date when there should be automated rotation -->
+            <column name="rotation_date" type="TIMESTAMP">
+                <constraints nullable="true"/>
```

Comment:
It may be simpler to keep this non-nullable and set the default to MAX in the future?

Not a big deal either way, but that would let you treat it as non-nullable in Kotlin.

Reply:
Trying this.

Reply:
Done